app/ide-desktop/client/src/index.ts

```diff
@@ -99,6 +99,8 @@ class App {
         this.setChromeOptions(chromeOptions)
         security.enableAll()
 
+        this.onStart()
```

Comment:
there might be a race condition as we do not `await` `onStart()`

Reply:
Also I didn't see here error handling, this could potentially close the IDE

Reply:
> there might be a race condition as we do not await onStart()

It should be safe to run those actions in the background

Reply:
> Also I didn't see here an error handling

Added error logging :+1: